config/manager/manager.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         image: controller:latest
         name: manager
         securityContext:
+          seccompProfile:
```

Comment:
that one will create issues with older kubernetes ...

Reply:
Yes, we need to update/create the docs with a minimum version in mind. This was causing problems for the e2e. We can change the tests, but I think we can afford to have this strict security set. wdyt?

Reply:
we would still break with OCP EUS :S